### PR TITLE
updated deprecated dependencies to latest version.

### PR DESCRIPTION
**Pull request by Alwaz**
Updated all deprecated dependencies to latest version.

### CODE REVIEW

**Comment by Alwaz** [comment]
Updated all deprecated dependencies to latest versions.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
-.vscode
\ No newline at end of file
+.vscode
+.next/
```
Comment (MrSlimCoder) [inline .gitignore]: it should be .next instead of .next/

**Comment by MrSlimCoder** [comment]
Please squash all commits for tracking.

**Review by MrSlimCoder — APPROVED** [review]
LGTM

**Comment by MrSlimCoder** [comment]
@all-contributors add @Alwaz for code.

**Pull request merged**